max_iter: 150
target:
  eta: 1
  num_agents: 1
  ub:
    - 4

source:
  eta: 1
  num_agents: 2
  ub:
    - 4
    - 4